source/install.rst

```diff
@@ -1 +1,297 @@
+``My Computer``).  You can do that by entering commands and pressing ``Enter``.
+Use the following commands:
+
+``dir``
+    Displays the contents of the current directory. For example, if the ``prompt``
+    shows  ``C:\Users\Name``, the ``dir`` command displays the contents of our home directory.
+
+``cd directory``
+    Changes the current directory. For example, if you are in ``C:\Users\Name``,
+    you can access the directory with your documents by entering ``cd Documents``. If you execute the
+    ``dir`` command, you will see something familiar.
+    The command  ``cd..`` will move you one level up in the directory tree, that is,
+    to the directory that contains your current directory.
+
+``mkdir directory``
+    Creates a new directory.
+
+
+Virtual environment
+-------------------
+
+A virtual environment is a tool that allows us to
+isolate our work from other parts of the system. This ensures that any
+new and experimental library you use will be confined to this sandbox and won't contaminate
+your system.
+
+To create a venv you need to choose a place where it will be created. For example, you can choose your home
+directory. If you select a different place, just remember where venv is created as we will use it often.
+
+In Windows, the path to your home directory, for the user  ``Ala``, will look like this:
+``C:\Users\Ala\``. In Linux, writing ``cd`` in the command line will get you to your $HOME directory.
+
+In Windows, to create virtual environment, write:
+
+.. code-block:: bat
+
+    C:\Users\Ala> python -m venv workshops
+
+This assumes that during installation python was added to your:w
+ PATAH.
+
+For Linux or Mac user:
+
+.. code-block:: sh
+
+    ~$ python3.6 -m venv workshops
+
+This creates a directory ``workshops``. This is your virtual environment. Right now it's still incative.
+
+To activate it on Windows write:
+
+.. code-block:: bat
+
+    C:\Users\Ala> workshops\Scripts\activate
+
+For Linux or Mac users:
+
+.. code-block:: sh
+
+    ~$ source workshops/bin/activate
+
+The ``python`` command will run the correct version of Python, so we will not have to enter the full
+path at the beginning nor the version at the end. Note ``(workshop)`` at the beginning of your new prompt.
+
+.. code-block:: bat
+
+    (workshops) C:\Users\Ala>where python
+    C:\Users\Ala\workshops\Scripts\python.exe
+    ...
+
+    (workshops) C:\Users\Ala>where pip
+    C:\Users\Ala\workshops\Scripts\pip.exe
+    ...
+
+    (workshops) C:\Users\Ala>python --version
+    3.6.5
+
+.. code-block:: sh
+
+    # Linux or Mac
+    (workshops) ~$ which python
+    /home/Ala/workshops/bin/python
+    (workshops) ~$ which pip
+    /home/Ala/workshops/bin/pip
+    ...
+
+    (workshops) ~$ python --version
+    3.6.0
+
+
+
+.. note::
+    You may already have the ``pip`` command available on your system. Always check which pip you are using with command: ``pip --version``.
```

Comment:
Maybe a short note that it will be explained what pip is in the next section? Or maybe even before the code where we check where it is. Because now it's like a magic word ;)